docs2/site/docs/migrations/migration8.md

```diff
@@ -246,6 +246,80 @@ At this time GraphQL.NET does not directly support the `MaxLength` and similar a
 implement your own attributes as shown above, or call the `Validate` method to set a validation
 function.
 
+### 6. Validation rules can read or validate field arguments and directive arguments
+
+Validation rules can now read or validate field arguments and directive arguments. This is useful
+for edge cases, such as when a complexity analyzer needs to read the value of a field argument
+to determine the complexity of the field. While it could also be used within custom validation rules
+to validate the value of field arguments or directive arguments, perhaps based on schema directives,
+it cannot be used to validate fields of input object graph types.
+
+Note that with the addition of `Validator` and `Parser` delegates, validation rules are no longer
+required to validate input values. If you plan to store validation rules within schema directives,
+such that they may be read by clients (see `ExperimentalFeatures.AppliedDirectives`), you may wish
+to consider adding a schema initialization visitor to walk the schema, applying the validation rules
+to the appropriate fields arguments via the `Validate` method, rather than using a validation rule
+to validate the field arguments. This will be much more efficient, as the validation rule will not
+be called for every field argument on every field, nor will it require a dictionary lookup to find
+the validation rule for the field argument.
```

Comment:
Add a note that both approach example is provided below, so the reader won't waste time trying to understand all the details.

Reply:
Done